betelgeuse.py

```diff
@@ -20,11 +20,17 @@
 import ssl
 import testimony
 import time
+import sys
 
 from collections import Counter
-from pylarion.exceptions import PylarionLibException
-from pylarion.work_item import TestCase, Requirement
-from pylarion.test_run import TestRun
+try:
+    from pylarion.exceptions import PylarionLibException
+    from pylarion.work_item import TestCase, Requirement
+    from pylarion.test_run import TestRun
+except ImportError as exception:
```

Comment:
I think Python will already do that by raising a proper ImportError.

What should be a really improvement is to add installation documentation, stating that Pylarion and its requirements should be installed in order to use Betelgeuse. We also have to take care about the information provided on the docs, since Pylarion is not a public project.